docs/Permissions/Onchain-Permissioning/Onchain-Permissioning.md

```diff
@@ -0,0 +1,53 @@
+description: Onchain Permissioning
+<!--- END of page meta data -->
+
+# Onchain Permissioning 
+
+Onchain permissioning uses smart contracts to store and maintain the node, account, and admin account whitelists. 
+Using onchain permissioning enables all nodes to read the whitelists from one location.                                        
```

Comment:
maybe 'source' is better than 'location'?

Reply:
Fixed